test/.eslintrc

```diff
@@ -15,3 +15,14 @@ rules:
   max-statements: 0
   no-unused-expressions: 0
   require-jsdoc: 0
+  max-len:
+    - 1
+    -
+      // https://regex101.com/r/hVU4sX/1
+      ignorePattern: '(^import\s([*])|([A-Za-z0-9]*)\s)|(\sfrom\s)'
```

Comment:
@cfpb/front-end-admins Would we want to add this `ignorePattern` for `import`s to our global `.eslintrc`?

Reply:
Removed `ignorePattern` and `ignoreStrings` from `.eslintrc` based on feedback from @ascott1